src/devices/bus/coco/coco_fdc.cpp

```diff
@@ -475,7 +285,7 @@ void coco_fdc_device_base::dskreg_w(u8 data)
 
 	// An email from John Kowalski informed me that if the DS3 is
 	// high, and one of the other drive bits is selected (DS0-DS2), then the
-	// second side of DS0, DS1, or DS2 is selected.  If multiple bits are
+	// second side of DS0, DS1, or DS2 is selected.	 If multiple bits are
```

Comment:
What exactly did you do to this space?

Reply:
Weird. I blame my tools. I replaced both spaces with a single space, because I am a single space after a full stop kind of guy.